kit.yml

```diff
@@ -151,25 +151,19 @@ credentials:
   openstack-cpi: *openstack
 
 provided:
-  aws: &provided_aws
-    aws:
-      keys:
-        secret_key:
-          prompt:    AWS Secret Key
-        access_key:
-          sensitive: false
-          prompt:    AWS Access Key
-
-  aws-cpi: *provided_aws
-
-  s3-blobstore:
-    blobstore/s3:
-      keys:
-        access_key:
-          sensitive: false
-          prompt:    AWS Blobstore Access Key
-        secret_key:
-          prompt:    AWS Blobstore Secret Key
```

Comment:
Did you mean to completely remove the S3 blobstore stuff?

Reply:
ah, you're right. I did a bad merge conflict resolution. putting it back. I did not mean to offend! ahhh

Reply:
lol no worries. I just thought maybe I misunderstood the rationale for removing the AWS portion and that that somehow included the S3 stuff.